pages/axelar-vm/overview.mdx

```diff
@@ -0,0 +1,88 @@
+# Axelar VM Architecture
+
+import { Callout } from 'nextra-theme-docs'
+
+This is a introduction to the architecture of the Axelar VM, first [introduced at the Interop Summit](https://axelar.network/blog/axelar-virtual-machine-future-of-interoperability). The Axelar VM is under active development, and will be released as parts are available to enable innovation across the industry. 
+
+There are three main groups that will benefit from the Axelar VM:
+* **Developers** - Think Interchain and more easily take your dApps further
+* **Integrators** - Connect your blockchain or tech stack to the Axelar Network
+* **Users** - Improved abstractions will make using web3 technologies simpler, safer, and easier
+
+## Foundation
+The Foundation of the Axelar VM is the ability to deploy and run smart contracts written in [CosmWasm](https://book.cosmwasm.com/). 
+
+<Callout emoji="💡">
+  Developers interested in trying out smart contract support on the Axelar Blockchain can give it a try on the [Axelar Wasm Devnet](https://bright-ambert-2bd.notion.site/Axelar-Devnet-885a966bf4764151a7e4c1fde68a04a2).
+</Callout>
+
+dApps built using CosmWasm are executed by the Axelar Network's [validators](https://axelarscan.io/validators), and are part of the state of the blockchain, creating an application layer on top of the existing consensus layer. Enterprising developers can use this application layer to begin to write connections to external infrastructure.
+
+## Interchain Amplifier
+The magic of this foundation of smart contracts manifests most strongly when integrators are able to plug new connections directly into the Axelar Network, allowing new paths, new chains, and more configurability for cross chain communication.
+
+We're designing paths for any integrator to create modules (contracts) deployed on Axelar that act as gateways to other ecosystems.
+
+### How Axelar previously integrated with blockchains
+To understand the future of our Interchain Amplifier, it can be helpful to understand how Axelar got to where we are today.
+
+Axelar has 30+ blockchain integrations that allow a combination of [message passing](/dev/general-message-passing/overview) and [token transfers](/dev/send-tokens/overview). Each of these blockchains requires implementation of technologies from several different parts of the stack. Today, Axelar validators execute code as part of the Axelar Node Software's consensus layer, requiring a full network upgrade (including downtime) for every modification or addition of a new chain.
```

Comment:
We don't necessarily need to upgrade the network when adding a new chain. If we are just adding another EVM chain or cosmos chain, that doesn't require a network upgrade. But if the chain we are adding requires a protocol update because it is unlike any of the existing chains we support, then we need to upgrade the network. I would say something like "requiring a full network upgrade for any change to the protocol"